models/inputs.yaml

```diff
@@ -59,7 +66,7 @@ inputs:
       with_entity_ids:
         - user
       with_columns:
-        - name: to_timestamp(created)
+        - name: created
```

Comment:
Hope this won't cause issue like [Cvent](https://github.com/rudderlabs/profiles_cvent/pull/11).

Reply:
The linked PR has changes in the `occurred_at_col` whereas this PR updates `with_columns`